src/cot/instruction_hooks.c

```diff
@@ -0,0 +1,59 @@
+#include <pmdsky.h>
+#include <cot/basedefs.h>
+#include <cot/logging.h>
+#include <cot/custom_instructions.h>
+
+// Loosely based on https://github.com/Adex-8x/jam-patches/blob/master/strung-up-by-sketches/CustomOpcodes/asm_patches/patch_ov36.asm#L137
+
+#if 0
+
+// const instead of #define so the constant can be referenced in Assembly
+__attribute((used)) static const int FIRST_CUSTOM_OPCODE = 0x1000;
```

Comment:
I'm a bit weirded out by the use of both `__attribute((used))` and `static` here. I assume this has been tested and it works, but is there a point in using them both together?